app/assets/javascripts/3dbio_viewer/src/webapp/components/dropzone/Dropzone.tsx

```diff
@@ -20,7 +20,7 @@ const Shade = styled.div<DropzoneRootProps>`
     display: flex;
     place-content: center;
     text-shadow: 1px 1px 10px black;
-    padding: 20px;
+    padding: 60px;
     align-items: center;
     border-width: 2px;
     border-radius: 2px;
```

Comment:
I am moving this code to the bottom (this came from existing code), to keep props and main component on the top of the file and the first thing you see.